react-app/src/components/ConfigEditor/ExtractorObject.js

```diff
@@ -345,7 +345,7 @@ function ExtractorObject(props) {
                 className="arg-input-width-limit"
               >
                 {arg.options.map((option) => (
-                  <option value={option} key={option}>
+                  <option value={option} key={option} selected={arg[arg.key] === option}>
```

Comment:
The console is throwing a warning on using the `selected` property on the individual options, which is strange since the property exists and seems to be working fine. Instead of using the `selected` property here, could we use the `defaultValue` prop in the parent `From.Select` component? Adding `defaultValue={arg[arg.key]}` should accomplish the same thing and keep the linter happy.